### PR TITLE
Created Side bar

### PR DESCRIPTION
**Pull request by niteshMeena1**

### CODE REVIEW

File: src/components/CoustomSideBar/CoustomSideBar.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import { cn } from "@/utils";
+import { Box } from "../Layout";
+import { VariantProps, cva } from "class-variance-authority";
+import { ComponentProps, forwardRef, useState } from "react";
+import { ProfileAvatar } from "../Avatar";
+
```
Comment (deeporiontechnology) [inline src/components/CoustomSideBar/CoustomSideBar.tsx:6]: black lines ??
only 1 line

File: src/components/CoustomSideBar/CoustomSideBar.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+
+
+interface sidebarItems {
```
Comment (deeporiontechnology) [inline src/components/CoustomSideBar/CoustomSideBar.tsx:10]: wrong naming conv

File: src/components/CoustomSideBar/CoustomSideBar.tsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  icon:string,
+  tabName:string
+}
+
+interface CoustomSideBarProps {
```
Comment (deeporiontechnology) [inline src/components/CoustomSideBar/CoustomSideBar.tsx:15]: Add I

File: src/components/CoustomSideBar/CoustomSideBar.tsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
+import { cn } from "@/utils";
```
Comment (deeporiontechnology) [inline src/components/CoustomSideBar/CoustomSideBar.tsx:1]: File name ??
CustomSideBar

Comment (niteshMeena1) [reply src/components/CoustomSideBar/CoustomSideBar.tsx:1]: can suggest me other name .

File: src/components/CustomSideBar/CustomSideBar.tsx [hunk 5]
```diff
@@ -0,0 +1,110 @@
+import { cn } from "@/utils";
+import { Box } from "../Layout";
+import { VariantProps, cva } from "class-variance-authority";
+import { ComponentProps, forwardRef, useState } from "react";
+import { ProfileAvatar } from "../Avatar";
+
+interface sidebarTabs {
```
Comment (deeporiontechnology) [inline src/components/CustomSideBar/CustomSideBar.tsx:7]: take care of names

**Pull request merged**